views/apartments/index.erb

```diff
@@ -1,5 +1,11 @@
+<h2>Apartments in Complex</h2>
+
 <ul>
-  <li><a href='/apartments/1'>Apartment 1</a></li>
-  <li><a href='/apartments/2'>Apartment 2</a></li>
-  <li><a href='/apartments/3'>Apartment 3</a></li>
+  <% @apartments.each do |apartment| %>
+    <li>
+      <a href="/apartments/<%= apartment.id %>">
+        <%= apartment.address %>
+      </a>
+    </li>
+  <% end %>
```

Comment:
nice job with the each method, you'll be using this enumerable frequently moving forward